### PR TITLE
Update protonvpn to 1.0.0 with stable URL

### PR DESCRIPTION
**Pull request by rob-patchett**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/protonvpn.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 cask 'protonvpn' do
   version '1.0.0'
-  sha256 '032d95003b2fb3c121a41dda736d4bc785ab6101eda65fb9e0b46c49487f6741'
+  sha256 '040da07ec2440a965cb2d1ce351c9041c3a407c16784c1ea1a63e0f47a57b050'
 
-  url "https://protonvpn.com/download/ProtonVPNv#{version.major_minor}.dmg"
```
[the remark this answers is not in the document]
Comment (rob-patchett) [reply Casks/protonvpn.rb]: Closing and re-opening with appcast's url

[the remark this answers is not in the document]
Comment (rob-patchett) [reply Casks/protonvpn.rb]: Have updated with URL from appcast

**Comment by commitay** [comment]
This is a URL change so the sha256 change is expected but here's VirusTotal for codesigning anyway.

> https://www.virustotal.com/#/file/040da07ec2440a965cb2d1ce351c9041c3a407c16784c1ea1a63e0f47a57b050/details

**Review by commitay — APPROVED** [review]

**Comment by commitay** [comment]
Thanks @rob-patchett!

**Comment by rob-patchett** [comment]
Thanks for the help 😄

**Pull request merged**